src/generic/codec.rs

```diff
@@ -12,11 +12,6 @@ use error::ProtocolError;
 
 /// Encodes and decodes gRPC message types
 pub trait Codec {
-    /// The content-type header for messages using this encoding.
-    ///
-    /// Should be `application/grpc+yourencoding`.
-    const CONTENT_TYPE: &'static str;
```

Comment:
Perhaps if this was replaced with a separate associated const on the `Encoder` trait, we could continue using it rather than hard-coding it?

Reply:
Done